plugin/environment.go

```diff
@@ -300,10 +301,14 @@ func (env *Environment) Shutdown() {
 		ap := value.(activePlugin)
 
 		if ap.supervisor != nil {
-			if err := ap.supervisor.Hooks().OnDeactivate(); err != nil {
-				env.logger.Error("Plugin OnDeactivate() error", mlog.String("plugin_id", ap.BundleInfo.Manifest.Id), mlog.Err(err))
+			//if err := ap.supervisor.Hooks().OnDeactivate(); err != nil {
+			//	env.logger.Error("Plugin OnDeactivate() error", mlog.String("plugin_id", ap.BundleInfo.Manifest.Id), mlog.Err(err))
+			//}
+			//ap.supervisor.Shutdown()
+			p, err := os.FindProcess(ap.supervisor.pid)
+			if err == nil {
+				p.Signal(syscall.SIGHUP)
```

Comment:
SIGHUP in servers is usually used these days to (gracefully) reload the config. See https://stackoverflow.com/a/690631 for a pretty good explanation.

To be clear, I think in this case we should be sending a SIGTERM first (maybe in Shutdown?), and then if it times out (i.e. the process is still there) send a SIGKILL.

Reply:
Added SIGTERM, and if error returned SIGKILL.  Would you prefer a time out check vs. error check?